packages/devtools_app/lib/src/screens/performance/panes/timeline_events/perfetto/_perfetto_web.dart

```diff
@@ -177,6 +177,7 @@ class _PerfettoViewController extends DisposableController
         'buffer': traceBinary,
         'title': 'DevTools timeline trace',
         'keepApiOpen': true,
+        'expandAllTrackGroups': true,
```

Comment:
this is the only change that needs reviewed. The remaining changes were generated by refreshing the perfetto build.